database/schema/ve_db_dev_data.sql

```diff
@@ -82,7 +90,16 @@ INSERT INTO cve_rule_mapping (cve_id, rule_id) VALUES
 (10, 5),
 (11, 5),
 (10, 6),
-(11, 6);
+(11, 6),
+(1, 8),
+(2, 8),
+(3, 9),
+(8, 10),
+(9, 10),
+(8, 11),
+(9, 11),
+(10, 12),
+(11, 12);
 
 INSERT INTO system_vulnerabilities (rh_account_id, system_id, cve_id, first_reported, when_mitigated, status_id, rule_id, rule_hit_details) VALUES
```

Comment:
Could you please "fix" testing data inserted here?
Afaik there are some records with `mitigation_reason` being NOT NULL. Can you please switch `rule_id` field for some of them to point to the new top level rules for passing?
There can be both, full rule id and the partial you've just implemented so nothing's wrong there. But it'd better for tests to have some records in `system_vulnerabilities` pointing to the "new partial" rules.